### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Jorgen-S-Engh**
Vennligst se gjennom mine endringer

### CODE REVIEW

File: homepage.html [hunk 1]
```diff
@@ -0,0 +1,185 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <link
```
Comment (eivindholvik) [inline homepage.html:7]: Her bruker du CDN. Oppgaven sier at du skal installere bootstrap, det gjør du ikke ved bruk av Content Delivery Network.

File: sass/stylesheet.scss [hunk 2]
```diff
@@ -1,3 +1,60 @@
...
+@import "component";
+
+
+.card{
+  border:none;
+}
+
+
+.profile-img{
+  border-radius: 50%;
```
Comment (SiriHoyas) [inline sass/stylesheet.scss]: There is a set of Bootstrap classes for border radius if you want to use that instead

File: sass/stylesheet.scss [hunk 3]
```diff
@@ -1,3 +1,60 @@
...
+
+
+}
+
+.bio-container{
+  max-width:300px;
+}
+
+.profile-section{
+  width:100%;
```
Comment (SiriHoyas) [inline sass/stylesheet.scss]: Could this be solved in Bootstrap with the w-100 class?

**Review by SiriHoyas — COMMENTED** [review]
The site looks good, there are some styles you have manually applied in SCSS that could maybe be done with Bootstrap (like the width of some elements maybe?).

After what I can tell, you have included everything that you should according to the brief. One thing to note is that Bootstrap is linked, as Eivind pointed out in the comment above.

Nice work!

**Pull request merged**